packages/golden-layout/src/items/Component.ts

```diff
@@ -66,7 +66,7 @@ export default class Component extends AbstractContentItem {
   }
 
   close() {
-    this.parent.removeChild(this);
+    this.parent?.removeChild(this);
```

Comment:
Trying to wrap my head around how this didn't cause problems before, but e2e tests would suggest that `Component` item types can be instantiated without a parent.